rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -189,13 +188,9 @@ public void remove(String state, URI lraId) {
             lras.remove(lraId);
         }
 
-        if (recoveringLRAs.containsKey(lraId)) {
-            recoveringLRAs.remove(lraId);
-        }
+        recoveringLRAs.remove(lraId);
 
-        if (locks.containsKey(lraId)) {
-            locks.remove(lraId);
-        }
+        locks.remove(lraId);
```

Comment:
was this part a general tidy up or was it related to JBTM-3243? if general I think it needs a separate issue?

Reply:
ditto

Reply:
I have revised on https://github.com/jbosstm/narayana/pull/1547#discussion_r365351134